projects/nae-example-app/src/app/doc/public-single-task-view/public-single-task-view.component.ts

```diff
@@ -0,0 +1,154 @@
+        new TaskResourceService(provider, config, fieldConverter, logger),
+        new PublicTaskResourceService(provider, config, fieldConverter, logger));
+};
+
+const caseResourceServiceFactory = (userService: UserService, sessionService: SessionService, authService: AuthenticationService,
+                                    router: Router, publicResolverService: PublicUrlResolverService,
+                                    provider: ResourceProvider, config: ConfigurationService) => {
+    return publicFactoryResolver(userService, sessionService, authService, router, publicResolverService,
+        new CaseResourceService(provider, config),
+        new PublicCaseResourceService(provider, config));
+};
+
+@Component({
+    selector: 'nae-app-public-single-task-view',
+    templateUrl: './public-single-task-view.component.html',
+    styleUrls: ['./public-single-task-view.component.scss'],
+    providers: [
+        TaskViewService,
+        PublicTaskLoadingService,
+        SearchService,
+        ChangedFieldsService,
+        {
+            provide: ProcessService,
+            useFactory: processServiceFactory,
+            deps: [UserService, SessionService, AuthenticationService, Router, PublicUrlResolverService, PetriNetResourceService,
+                PublicPetriNetResourceService, LoggerService]
+        },
+        {
+            provide: TaskResourceService,
+            useFactory: taskResourceServiceFactory,
+            deps: [UserService, SessionService, AuthenticationService, Router, PublicUrlResolverService,
+                LoggerService, ResourceProvider, ConfigurationService, FieldConverterService]
+        },
+        {
+            provide: CaseResourceService,
+            useFactory: caseResourceServiceFactory,
+            deps: [UserService, SessionService, AuthenticationService, Router, PublicUrlResolverService,
+                ResourceProvider, ConfigurationService]
+        },
+        {
+            provide: NAE_BASE_FILTER,
+            useFactory: publicBaseFilterFactory,
+            deps: [Router, ActivatedRoute, ProcessService, CaseResourceService, SnackBarService, TranslateService, PublicTaskLoadingService]
+        },
+        {
+            provide: AllowedNetsService,
+            useFactory: localAllowedNetsServiceFactory,
+            deps: [AllowedNetsServiceFactory, ActivatedRoute]
+        },
+        {   provide: NAE_VIEW_ID_SEGMENT, useValue: 'publicView'},
+        {   provide: AllowedNetsServiceFactory, useClass: AllowedNetsServiceFactory},
+        ViewIdService,
+        {provide: TaskContentService, useClass: SingleTaskContentService},
+        TaskDataService,
+        FinishTaskService,
+        TaskRequestStateService,
+        TaskEventService,
+        {provide: NAE_TASK_OPERATIONS, useClass: SubjectTaskOperations},
+    ]
+})
+export class PublicSingleTaskViewComponent extends AbstractSingleTaskViewComponent implements AfterViewInit {
```

Comment:
Plz  test

Reply:
There is a test for this example component, and hence there is no other functionality in that component except the creation of templates and the component itself, the coverage is pretty high in that file.